game_eggs/vintage_story/egg-vintage-story.json

```diff
@@ -1,13 +1,18 @@
 {
     "_comment": "DO NOT EDIT: FILE GENERATED AUTOMATICALLY BY PTERODACTYL PANEL - PTERODACTYL.IO",
     "meta": {
-        "version": "PTDL_v1"
+        "version": "PTDL_v2",
+        "update_url": null
     },
-    "exported_at": "2020-03-20T23:54:00+01:00",
-    "name": "Vintage Story",
-    "author": "mail@wuffy.eu",
+    "exported_at": "2023-03-08T21:10:40+00:00",
+    "name": "Vintage Story - Custom",
```

Comment:
You can not change the email nore the name of the egg

Reply:
Forgot to change that one back; was testing on the side and email was forcefully changed